nav2_msgs/msg/BinaryState.msg

```diff
@@ -0,0 +1,2 @@
+# Binary state
+bool state
```

Comment:
`std_msgs/Bool`?

Reply:
They are officially recommend to have a new message for that and seems deprecated since Foxy:
```
cat ros2/common_interfaces/std_msgs/msg/Bool.msg
# This was originally provided as an example message.
# It is deprecated as of Foxy
# It is recommended to create your own semantically meaningful message.
# However if you would like to continue using this please use the equivalent in example_msgs.

bool data
```
We have also `ros2/example_interfaces/msg/Bool.msg` which was not deprecated, but still recommending to make your own service. I'd like to agree, that utilizing an existing service is better to reinvent the wheel. But will it be OK, if we will use `example_interfaces` as a messages utilized by Nav2?

Reply:
https://github.com/ros-planning/navigation2/search?p=2&q=std_msgs

We have hundreds of uses of std_msgs, I doubt that'll be actually deprecated in another 5 years. What's 1 more?

Reply:
Hm... Despite the `std_msgs/msg/Bool.msg` is claimed to be deprecated in ROS2, I see no build or other issues after I've changed it. So, the https://github.com/ros-planning/navigation2/pull/3228/commits/07ada1f2212291a86592bd0b4399bd5d1e1998f0 does this.